wolox-books/src/app/services/user.service.ts

```diff
@@ -22,4 +22,16 @@ export class UserService {
     let response = this.http.post(this.root_url, {user: user }, {headers: headers})
       .subscribe(a => this.Router.navigate(['login']));
   }
+
+  logIn(login) {
+
+    const headers = new HttpHeaders({
+      'Content-Type': 'application/json'
+    });
+    
+    const url = this.root_url + '/sessions';
+
+    let response = this.http.post(url, {session: login }, {headers: headers})
```

Comment:
if the object has this format: `{headers: headers}` you can do this: `{ headers }`.

also every time you create an object for example the `{session: login }` add a space at the begining and at the end of the object like this: `{ session: login }`